### PR TITLE
Fix for Active Record callbacks

### PR DESCRIPTION
**Pull request by fabn**
Will solve #138

### CODE REVIEW

**Comment by orlando** [comment]
@fabn can you add a test for this?. Testing is something we need to improve. If it's not that difficult, it will be great to have some test coverage of this code

**Comment by fabn** [comment]
@orlando I did that and much more but I din't receive any feedback. Could you please take a look into this?

Thanks.

**Comment by orlando** [comment]
@fabn Awesome!. I'll take a look today.

Thank you for your patience

File: spec/dummy/app/assets/javascripts/application.js [hunk 1]
```diff
@@ -2,27 +2,29 @@
...
+// Read Sprockets README (https://github.com/rails/sprockets#sprockets-directives) for details
+// about supported directives.
 //
 //= require jquery
 //= require jquery_ujs
-//=
-//= require jquery.ui.widget
-//= require jquery.iframe-transport
-//= require jquery.fileupload
+//= require bootstrap-sprockets
```
Comment (orlando) [inline spec/dummy/app/assets/javascripts/application.js:15]: @fabn I don't think we should introduce this dependency, let's keep the dummy app as barebones as possible

**Comment by sharq1** [comment]
@orlando a merge, maybe?

**Comment by orlando** [comment]
Hey @sharq1,

Sorry, I used to use this library in production but not anymore. I don't think I can review this change since is so big and includes things that are not necessary to have in the main repo (example `bootstrap-sprockets`)

File: spec/dummy/app/assets/javascripts/application.js (hunk 1 above)
Comment (fabn) [reply spec/dummy/app/assets/javascripts/application.js:15]: @orlando that's a devo dependency that is not pulled when you add this gem as dependency. If I remember it's used in integration tests as js library to write some end to end tests but honestly I wrote this long time ago... I think is good to have e2e specs and dev dependency won't hurt gem itself.

**Comment by fabn** [comment]
I'm still using this gem (my fork) in a couple of production projects on rails 5.2 but now that activestorage is out and cloudinary support that natively I don't know if is still worth to maintain this gem. However if you're interested in merging this I can collaborate in my spare time.

**Pull request closed without merging**